### PR TITLE
A few updates post eslint

### PR DESCRIPTION
**Pull request by gabrieldutra**
## What type of PR is this? (check all applicable)

- [x] Other

## Description
@arikfr here are a few updates I had in mind (was quicker to share in a PR than in comments). The ones @kravets-levko mentioned I'm not so sure about, so will confirm them first.

### CODE REVIEW

File: client/app/components/EditParameterSettingsDialog.jsx [hunk 1]
```diff
@@ -86,13 +86,13 @@ function EditParameterSettingsDialog(props) {
 
   // fetch query by id
   useEffect(() => {
-    const { queryId } = props.parameter;
+    const queryId = props.parameter.queryId;
```
Comment (arikfr) [inline client/app/components/EditParameterSettingsDialog.jsx]: Is this change needed for eslint to detect the correct value for the dependencies array?

Comment (gabrieldutra) [reply client/app/components/EditParameterSettingsDialog.jsx]: Yes, using it like this it automatically corrects to `[props.parameter.queryId]`

**Comment by gabrieldutra** [comment]
@kravets-levko ath else you think it's important here?
